lib/kazoo/cluster.rb

```diff
@@ -33,6 +33,13 @@ def brokers
       end
     end
 
+    def consumergroups
+      @consumergroups ||= begin
+        consumers = zk.get_children(path: "/consumers")
```

Comment:
`children` here might come back with `nil`, so we should totally check for that.